django/text_extractor/views.py

```diff
@@ -213,9 +206,7 @@ def download_document(request, file_id, user_request_id):
     user_request = UserRequest.objects.get(access_key, id=user_request_id)
 
     try:
-        output_file = user_request.output_files.get(
-            access_key=access_key, file_id=file_id
-        )
+        output_file = user_request.output_files.get(access_key=access_key, id=file_id)
```

Comment:
Since there is no special "file_id" field anymore, we just use the normal ID inherited from SecureModel